source/guide_snibox.rst

```diff
@@ -0,0 +1,216 @@
+.. highlight:: console
+
+.. author:: Maximilian F. Kapfer <maximilian.kapfer@gmail.com>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/snibox.png
+      :align: center
+
+######
+Snibox
+######
+
+Snibox is a self-hosted snippet manager. Developed to collect and organize code snippets. Supports various programming languages, markdown, plain text.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PostgreSQL_
+  * Ruby_
+  * supervisord_
+  * domains_
+
+
+License
+=======
+
+Snibox is released under the MIT License. All relevant legal information can be found here:
+
+  * https://opensource.org/licenses/MIT
+
+
+Prerequisites
+=============
+For this service we need to use Ruby in the version 2.6 and PostgreSQL in the version 12.
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version list ruby
+    - 2.5
+    - 2.6
+    - 2.7
+    - 3.0
+
+ [isabell@stardust ~]$ uberspace tools version use ruby 2.6
+    Selected ruby version 2.6
+    The new configuration is adapted immediately. Patch updates will be applied automatically.
+
+ [isabell@stardust ~]$ uberspace tools version list postgresql
+    - 10
+    - 11
+    - 12
+    - 13
+
```

Comment:
```suggestion
```
The list of versions will change in the future, making this section confusing. The available versions are listed in the manual, which is linked above.